assignments/prototypes.js

```diff
@@ -15,13 +15,29 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+const GameObject (){
+  this.createdAt: "Date()",
+  this.name: ,
+  this.dimensions: ,
```

Comment:
When you are using constructor functions, you do not use object syntax. Instead of writing `this.dimensions: value,`, you will write `this.dimensions = value;`